CoordImporter/Plugin.cs

```diff
@@ -46,14 +36,14 @@ public sealed class Plugin : IDalamudPlugin
             PluginInterface = pluginInterface;
             this.CommandManager = commandManager;
             Chat = chat;
-            DataManager = dataManager;
             Logger = logger;
+            DataManagerManager = new DataManagerManager(dataManager);
 
             HuntHelperManager = new HuntHelperManager();
 
             // Invalidate the Placename cache because the sheet in here may not correspond to the correct ClientLanguage.
```

Comment:
These lines need to go with the data manager stuff for loading the map data

Reply:
NVM, you are using `PlaceName.Row` and `Map.Row` in the map data loading method, so this stops lumina from loading the localized data into the non-localized parent table (thus causing mayhem).

Reply:
ohh it was because of that! should i take this out, then?